### DISCUSSION TITLE
I love achievements

### DISCUSSION THREAD

**Discussion by cj2001** [community/community#20565] [Other Feature Feedback, Questions, & Ideas]
That is all.  I love them.

Kudos from an ex-Hubber!  ❤️

**Comment by Mehul2203** [comment]
Hahaha, me too!!!

**Comment by RavelloH** [comment]
Indeed, it's really interesting!

**Comment by hvsw** [comment]
I came here to say that it's pretty nice, congrats on the idea!

**Comment by faelfernandes** [comment]
It really is a really cool thing!

**Comment by thuongtruong1009** [comment]
Hi @cj2001 
Well, it's great that you like all the achievement badges. I'm just like you, so I'm always working to find and improve my achievements. Here are the badges that I earned.
![image](https://user-images.githubusercontent.com/71834167/180589153-a6fd684c-3504-4691-b97d-26b177fc5d67.png)

See more at [my profile](https://github.com/thuongtruong1009)
Thank you!

**Comment by WilczekCK** [comment]
Its cool, but imo the images of them are kinda ugly 👎

**Comment by rlaope** [comment]
that is so cool

**Comment by JackHowa** [comment]
I'm into it! Helps you feel like you're a part of OSS more!

**Comment by vishnu322010326030** [comment]
that's awesome....

**Comment by ITZU-MYK-OFFICIAL** [comment]
That's so cool.I like it👌

**Comment by PedrodoDivino** [comment]
Eu gostei bastante dessa forma de gameficar o que fazemos no github. Parabéns!

**Comment by tamilarasana** [comment]
Hahaha, me too!!!
